persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -9,9 +18,99 @@ CREATE TABLE IF NOT EXISTS users
     first_name      VARCHAR(36) NOT NULL,
     last_name       VARCHAR(36) NOT NULL,
     privilege_level VARCHAR(16) NOT NULL,
-    password_hash   BYTEA       NOT NULL
+    country_id      BIGINT,
+    password_hash   BYTEA       NOT NULL,
+
+    CONSTRAINT fk_user_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS schools
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    name       VARCHAR(36) NOT NULL,
+    address    VARCHAR(64) NOT NULL,
+    hidden     BOOLEAN              DEFAULT FALSE,
+    country_id BIGINT,
+
+    CONSTRAINT fk_school_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS school_additional_contacts
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    school_id  BIGINT      NOT NULL,
+    name       VARCHAR(36) NOT NULL,
+    email      VARCHAR(36),
+    address    VARCHAR(36),
+    phone      VARCHAR(36),
```

Comment:
What if this was just a join table to `users`? Between user id and school id? That way we're not duplicating name and email, and address/phone are optional anyways.

Reply:
Would we want to create users just to hold contact information? In my mind, `user` implies the ability to login, where "contact" just holds information?

Reply:
True, if Amanda is able to insert contacts herself, not all of them will be users. But if those users then make accounts, wouldn't we want to associate the users with the contacts? Especially if the users are inputting data, it'd be good to know that they are a contact as well, and that relationship could be stored within a join table.

Reply:
Ahh... in my mind I envisioned these "contacts" as like school principal or some education district person--in which case I figured these would never really overlap with users.
So you think we should do `users` purely for login, `contacts` for personal information, and like a junction table to join `users` and `contacts`?